### DISCUSSION TITLE
Dreambooth working as before (very good)

### DISCUSSION THREAD

**Discussion by Bullseye-StableDiffusion** [TheLastBen/fast-stable-diffusion#1614] [Show and tell]
I have finally made it to work as before.
Check out this notebook: https://github.com/Bullseye-StableDiffusion/fixed/blob/main/fast_DreamBooth_fixed.ipynb
Took me some good hours to make it to work, because I'm no coder. Used my logical thinking to revert anything to a stable version. I don't know how to fork a certain commit, so I downloaded the older commit and uploaded on a new repository and modified everything accordingly.
I was right about what I was saying. Some changes from these days "destroyed" the notebook.

Edit: No more distortions and weird images. Add the name of the .jpg files in the prompts to get more characteristics of your character.

**Comment by TheLastBen** [comment]
You didn't change anything in that notebook

**Reply by Bullseye-StableDiffusion** [reply]
I changed the versions of the dependencies to be like in that older commit.

**Reply by Bullseye-StableDiffusion** [reply]
> I changed the versions of the dependencies to be like in that older commit.

Edit: I'm no coder, but at least it works as before now.

**Reply by Bullseye-StableDiffusion** [reply]
Just wishing to help the community, bro. I have nothing to win anyway. I was annoyed by that issue and tried to find a solution.

**Comment by moxumbic** [comment]
working good

**Reply by Bullseye-StableDiffusion** [reply]
Thanks for confirming!

**Comment by meunumerotim333** [comment]
Thaank you <3 

That's what I was requesting. Just to have older method like was before. older is better. We don't need to mess on what is working.

It's good to optmize but always needs to test if the quality stills. If something keep worse don't make sense to change.

**Reply by Bullseye-StableDiffusion** [reply]
Exactly. I tested almost every update.
